include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -79,6 +79,18 @@ class all_view
         cgh.require(m_acc);
     }
 
+  private:
+    accessor_t create_accessor(sycl::buffer<_T, 1> __buf, diff_type __offset, diff_type __n)
```

Comment:
Uglification: `__create_accessor`. Actually this comment is applicable to many things in `all_view`: `__accessor_t`, `m_acc`, `return_t`, etc.

Reply:
Actually, I tried wo be consistent with whole `class all_view`.